tools/integration_tests/managed_folders/test_helper.go

```diff
@@ -112,14 +112,18 @@ func createDirectoryStructureForNonEmptyManagedFolders(ctx context.Context, stor
 	if err != nil {
 		log.Fatalf("Failed to clean up test directory: %v", err)
 	}
-	client.CreateManagedFoldersInBucket(ctx, controlClient, path.Join(testDir, ManagedFolder1), bucket)
 	f := operations.CreateFile(path.Join("/tmp", FileInNonEmptyManagedFoldersTest), setup.FilePermission_0600, t)
 	defer operations.CloseFile(f)
-	operations.CopyFileInBucket(path.Join("/tmp", FileInNonEmptyManagedFoldersTest), path.Join(testDir, ManagedFolder1), bucket, t)
+	managedFolder1 := path.Join(testDir, ManagedFolder1)
+	managedFolder2 := path.Join(testDir, ManagedFolder2)
+	simulatedFolderNonEmptyManagedFoldersTest := path.Join(testDir, SimulatedFolderNonEmptyManagedFoldersTest)
+
+	client.CreateManagedFoldersInBucket(ctx, controlClient, path.Join(testDir, ManagedFolder1), bucket)
+	client.CopyFileInBucket(ctx, storageClient, path.Join("/tmp", FileInNonEmptyManagedFoldersTest), path.Join(managedFolder1, FileInNonEmptyManagedFoldersTest), bucket)
 	client.CreateManagedFoldersInBucket(ctx, controlClient, path.Join(testDir, ManagedFolder2), bucket)
-	operations.CopyFileInBucket(path.Join("/tmp", FileInNonEmptyManagedFoldersTest), path.Join(testDir, ManagedFolder2), bucket, t)
-	operations.CopyFileInBucket(path.Join("/tmp", FileInNonEmptyManagedFoldersTest), path.Join(testDir, SimulatedFolderNonEmptyManagedFoldersTest), bucket, t)
-	operations.CopyFileInBucket(path.Join("/tmp", FileInNonEmptyManagedFoldersTest), testDir, bucket, t)
+	client.CopyFileInBucket(ctx, storageClient, path.Join("/tmp", FileInNonEmptyManagedFoldersTest), path.Join(managedFolder2, FileInNonEmptyManagedFoldersTest), bucket)
```

Comment:
Shouldn't path be `path.Join(testDir, ManagedFolder2, FileInNonEmptyManagedFoldersTest)` ?

Reply:
Essentially that is the path being taken, I just created a variable 
`managedFolder2 := path.Join(testDir, ManagedFolder2)
 ......path.Join(managedFolder2,FileInNonEmptyManagedFoldersTest) `